templates/test/ci/patches/cluster-label-azuredisk-csi-driver.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: ${CLUSTER_NAME}
+  labels:
+    azureDiskCSI: "true"
```

Comment:
since we need a separate label patch for each addon, let's rename the calico one from the generic `cluster-label.yaml` to something more specific?